httpd/handler.go

```diff
@@ -164,7 +164,7 @@ func (h *Handler) serveWrite(w http.ResponseWriter, r *http.Request, user *influ
 	for {
 		if err := dec.Decode(&br); err != nil {
 			if err.Error() == "EOF" {
-				w.WriteHeader(http.StatusOK)
+				w.WriteHeader(http.StatusCreated)
 				return
```

Comment:
If we get EOF, it means that at least one thing was created, otherwise it would of errored out somewhere else in this routine.  This line can only be hit if we did get valid json, wrote it, and now have nothing else to do.  EOF in this case actually means "success".

If anything else in this routine fails, it will exit and write the proper error.